### ISSUE TITLE
npm package fails to run in Debian 10

### ISSUE THREAD

**Issue by msanguineti** [dprint/dprint#483]
Hello, I couldn't find a report for this (if it exists, I am really sorry to re-report it).

my specs:

Debian 10
node 16

I have installed `dprint` through npm `npm i -D dprint` and i get this error:

```
npx dprint help
/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.29' not found
```

From experience, I believe that `dprint` was compiled under an ubuntu 20 machine (or some 'latest' version of some distro)? If this is the case, could you please instruct your CI to build on a older LTS Ubuntu or Debian version which is still very much active?

Thank you very much for your time.

**Comment by dsherret** [comment]
Sorry, I didn't notice this issue until just now. I tried running the CI on ubuntu-18.04 and produced a release artifact. Would you be able to do me a favour and download it, try it out, and see if that fixes the issue for you?

https://github.com/dprint/dprint/actions/runs/1977780143 (see "linux-artifacts" at the bottom of this CI run... extract the zip then try out the binary by running `./dprint help`)

**Comment by msanguineti** [comment]
> Sorry, I didn't notice this issue until just now. I tried running the CI on ubuntu-18.04 and produced a release artifact. Would you be able to do me a favour and download it, try it out, and see if that fixes the issue for you?
> 
> https://github.com/dprint/dprint/actions/runs/1977780143 (see "linux-artifacts" at the bottom of this CI run... extract the zip then try out the binary by running `./dprint help`)

Yep, it's alive!

I tell you, this `*-latest` came 'round to bit me, also I've reported in other OS projects... always think of the **earliest** LTS

**Comment by dsherret** [comment]
@msanguineti thanks for testing! This is resolved in 0.23.1 and should be available now.
